showcase.html

```diff
@@ -271,6 +251,28 @@ <h3>Anxious Software Inc.
                         Virtual Presenter Pro is a virtual reality based tool that helps people to grow more confident with public speaking and improve their public speaking skills.
                     </p>
                 </div>
+                <div class="project">
+                    <h2>RelaxYourMind</h2>
```

Comment:
Can we move this further up so that it's sorted alphabetically by the project name?

Reply:
fixed in fa87b0f61cbfdf61c9eb38d6681ef1d1e2063110